primateid/resources/scripts/primateid/triggers.js

```diff
@@ -68,11 +68,18 @@ exports.init = function (EHR) {
             filterArray: [LABKEY.Filter.create('participantid', row['participantid'], LABKEY.Filter.Types.EQUAL)],
             requiredVersion: 17.1,
             success: function (data) {
+
+                // Getting the participantId field is the normal case, but in auto-tests we need to grab the Id field
+                var participantid = row['participantid'];
+                if (!participantid) {
+                    participantid = row['Id'];
+                }
```

Comment:
var participantid = row['participantid'] || row['Id'];

Reply:
These triggers use really old JS. I'm not sure it even recognizes || in assignment. Just to be safe I'm going to leave this.